### DISCUSSION TITLE
PD Image viewer plugin (imagine, XnView etc) (pdi format)

### DISCUSSION THREAD

**Discussion by hi5** [tannerhelland/PhotoDemon#570] [General]
I suspect PD users might find it helpful it there would be PD plugins available for popular image viewers such as

* [Imagine](https://www.nyam.pe.kr/dev/imagine/), [Plugins SDK)](https://www.nyam.pe.kr/dev/imagine/download.php?dir=plugin&name=Plugin_SDK&version=1.5.3) - C++ (I think)
* [XnView](https://www.xnview.com/en/), [Plugins SDK](https://www.xnview.com/download/plugins/PluginsSDK-win.zip) - C
* [IrfanView](https://www.irfanview.com/), you need to contact the dev. for details, [see bottom of page](https://www.irfanview.com/plugins.htm)
* no doubt many others.

Personally I would like to see one form Imagine :) 🤞 someone will pick up this challenge at some point.

**Comment by tannerhelland** [comment]
Thank you for this suggestion, @hi5 !  I especially appreciate the links to existing resources.

I have no idea where to begin with this one, but I would be happy to help any other developers brave enough to tackle it.

**Reply by selvakannan** [reply]
pdi format supported thumbnailer attached to tanner helland's photodemon code with pure vb6.
link is ..https://github.com/selvakannan/Photodemon_Thumbnailer1

**Reply by hi5** [reply]
Please create **one** (1) post, not several (you made 6 posts) and do no reply to messages that don't have anything to do with your suggestion.
My request is **not** about Photodemon, but image viewers (other programs) in general that use plugins to be able to display various graphic file formats.

**Comment by selvakannan** [comment]
pdi format supported thumbnailer attached to tanner helland's photodemon
code with pure vb6.
link is ..https://github.com/selvakannan/Photodemon_Thumbnailer1

**Reply by hi5** [reply]
Sure, but it has nothing to do with **my** request - my request is **not about Photodemon**.  I'm going to block you so I no longer see your posts.
